src/assets/SubstanceList.jsx

```diff
@@ -0,0 +1,351 @@
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 10,
+    name: "DMT",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 11,
+    name: "2C-B",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 12,
+    name: "Benzodiazepines",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 13,
+    name: "Cafeine",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 14,
+    name: "GHB",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 15,
+    name: "Opioids",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 16,
+    name: "Alcohol",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
```

Comment:
You can create a constant file called possibleResults and define there the possible results, something like:

const possibleResults = {
  synergy: {
    name: 'Synergy',
    icon: SynergyIcon
  },
  ...
}

Reply:
should this const be in the substanceCombination component?

Reply:
Same as the comment above, maybe we can have 'synergy' as the `id`. And you can create a `constants` folder in the `src` project and there create a `substances.js` file to create and export the `possibleResults` from there. We can do this in case we need to use it in another place of the app.